py/mpconfig.h

```diff
@@ -1643,6 +1643,13 @@ typedef double mp_float_t;
 #endif
 #endif
 
+// Explicitly annotate switch case fall throughs
+#if defined(__GNUC__) && __GNUC__ >= 7
```

Comment:
How about enabling `-Wimplicit-fallthrough` for clang as well? See http://clang.llvm.org/docs/AttributeReference.html#fallthrough

Reply:
Yeah, I tried that in the first three attempts, but it turns out clang only does -Wimplicit-fallthrough for C++

Reply:
Aha ok, wasn't clear to me.

Reply:
No, me neither